corehq/apps/api/tests/lookup_table_resources.py

```diff
@@ -249,6 +249,23 @@ def test_update_field_name(self):
         data_type = LookupTable.objects.get(id=self.data_type.id)
         self.assertEqual(data_type.fields[0].field_name, 'property')
 
+    def test_update_fails_with_two_field_names(self):
+        lookup_table = {
+            "fields": [{"name": "property", "field_name": "prop"}],
+            "tag": "lookup_table",
+        }
+
+        response = self._assert_auth_post_resource(
+            self.single_endpoint(self.data_type.id), json.dumps(lookup_table), method="PUT")
+        self.assertEqual(response.status_code, 400)
+        errors = json.loads(response.content.decode("utf-8"))
+        print(errors)
```

Comment:
nit: print

Reply:
I left these in because they are very helpful when debugging test failures.